src/pages/workspaces/workspace/analysis/tool-utils.ts

```diff
@@ -160,6 +180,10 @@ export const allAppTypes: AppToolLabel[] = _.flow(_.map('appType'), _.compact)(a
 
 export const isPauseSupported = (toolLabel: ToolLabel): boolean => !_.find((tool: Tool) => tool.label === toolLabel)(tools)?.isPauseUnsupported
 
+export const isSettingsSupported = (toolLabel: ToolLabel): boolean => !_.find((tool: Tool) => tool.label === toolLabel)(tools)?.isSettingsUnsupported
+
+export const getToolDisplayName = (toolLabel: ToolLabel): String => !!toolLabelDisplays[toolLabel] ? toolLabelDisplays[toolLabel] : toolLabel
```

Comment:
This function isn't necessary - just use `toolLabelDisplays[toolLabel]` in the location you want the display value.
Because of TypeScript, any ToolLabel MUST have a record in 'toolLabelDisplays.'

Reply:
This makes sense. Here's the fix - this will also fix a tiny bug as well.
In ContextBar.test.js:
Line 222 - change `tool: 'Azure'` to `tool: JupyterLab` 
Line 433 - change `expect(getByLabelText(new RegExp(/Azure Environment/i)))` to `expect(getByLabelText(new RegExp(/JupyterLab Environment/i)))`

Reply:
The reason it had been Azure is because the backend stores the tool as 'Azure' but we adjust it to 'JupyterLab' in Runtimes.js on listV2. It was part of the transition away from using Azure as the tool without having to migrate the backend.

Reply:
Ah so it is a bug. I will change that and remove `getToolDisplayName` function. Thank you!